### PR TITLE
removed '_' in modal title for medical_problem

### LINKED ISSUE #1568: remove underscore in section add "medical problem"
Steps to Reproduce the Issue:

1. Go to a patient profile.
2. using Patient Finder (“Patient/Client” from the main menu > “Finder” under “Patient/Client”) and choose a patient.
3. Select “Issues” under the patient’s name.
4. Create any issue by clicking the “Add" next to "medical problem"
The title of the modal contains an underscore

### PR DESCRIPTION
**Pull request by MathurinNkenfack**
Closes #1568

### CODE REVIEW

File: interface/patient_file/summary/stats_full.php [hunk 1]
```diff
@@ -100,6 +100,9 @@ function dopclick(id, category , type) {
 
     // function to open izi-modal
     function  initIziLink(link , width , height, category, type) {
+        if(category == "medical_problem"){
+            category = "medical problem";
```
Comment (muarachmann) [inline interface/patient_file/summary/stats_full.php]: Can you capitalize this by the way what of a generic function to strip the underscores and replace with space then capitalize. That should work for all the categories.

Comment (MathurinNkenfack) [reply interface/patient_file/summary/stats_full.php]: ok

File: interface/patient_file/summary/stats_full.php [hunk 2]
```diff
@@ -97,9 +97,20 @@ function dopclick(id, category , type) {
         alert("<?php echo addslashes(xl('You are not authorized to add/edit issues')); ?>");
         <?php endif; ?>
     }
-
+    //function to remove a character
+    function strReplace(str){
+        var newstr = str.replace('/_/g', " ");
```
Comment (muarachmann) [inline interface/patient_file/summary/stats_full.php]: Why do I still see the strReplace function

**Comment by MathurinNkenfack** [comment]
#1568

File: interface/patient_file/summary/stats_full.php [hunk 3]
```diff
@@ -97,9 +97,17 @@ function dopclick(id, category , type) {
...
+  
+        //function to remove underscore
+    function titleFormat(str){
+        var newstr = str.replace(/_/g, " ");
+        return newstr.toUpperCase();
+    }
     // function to open izi-modal
     function  initIziLink(link , width , height, category, type) {
+
+        category = titleFormat(category);
```
Comment (muarachmann) [inline interface/patient_file/summary/stats_full.php]: LGTM

**Pull request merged**